### PR TITLE
Add additional arguments to Downtime struct

### PR DESCRIPTION
**Pull request by vanvlack**
re: #226 

Adds additional arguments to the `Downtime` struct

### CODE REVIEW

**Comment by vanvlack** [comment]
@ojongerius do you have a chance for review?

**Comment by bkabrda** [comment]
@vanvlack hey, is there any reason why there are no getters/setters for `Downtime.MonitorTags`? This PR looks great otherwise BTW!

**Comment by vanvlack** [comment]
> is there any reason why there are no getters/setters for `Downtime.MonitorTags`

Unsure, will try to regenerate the get/set stuff and see what happens.

**Comment by vanvlack** [comment]
@bkabrda by the looks, no arrays (even ones elsewhere) are in the generated get/set file.

File: integration/downtime_test.go [hunk 1]
```diff
@@ -98,13 +98,14 @@ func getTestDowntime() *datadog.Downtime {
...
-		Disabled:   datadog.Bool(false),
-		Message:    datadog.String("Test downtime message"),
-		Scope:      []string{"env:downtime_test"},
-		Start:      datadog.Int(1577836800),
-		End:        datadog.Int(1577840400),
-		Recurrence: r,
+		Active:      datadog.Bool(false),
+		Disabled:    datadog.Bool(false),
+		Message:     datadog.String("Test downtime message"),
+		MonitorTags: []string{"some:tag"},
```
Comment (ojongerius) [inline integration/downtime_test.go]: Could you please add `ParentId` too?

File: integration/downtime_test.go [hunk 2]
```diff
@@ -98,13 +98,14 @@ func getTestDowntime() *datadog.Downtime {
...
+		Scope:       []string{"env:downtime_test"},
```
Comment (ojongerius) [inline integration/downtime_test.go]: Ah and `TimeZone`

**Comment by bkabrda** [comment]
@vanvlack ah, right, the getters and setters are only generated for struct members that are pointers, so everything is correct in this case.

**Comment by bkabrda** [comment]
Hey @vanvlack if you could address comments from @ojongerius and rebase, that would be awesome and also allow me to merge this PR.

**Comment by vanvlack** [comment]
@bkabrda updated

**Comment by bkabrda** [comment]
Ok, the tests are passing and all comments that @ojongerius made were addressed. LGTM, merging. Thanks a lot for the PR!

**Pull request merged**